src/offline.njk

```diff
@@ -0,0 +1,15 @@
+---
+layout: layouts/error.njk
+permalink: /offline.html
+eleventyExcludeFromCollections: true
+dynamicPermalink: false
+templateClass: offline-page
+title: Offline
+---
+<section class="offline-section">
+  <div class="offline-info">
+    <h1>Jste Offline</h1>
+    <p>To je sice trochu problém, ale tenhle web by měl trochu fungovat i&nbsp;tak. Co takhle zkusit to zatím třeba znovu
```

Comment:
**nitpick (non-blocking)**: Are those `&nbsp;`s IDE created? I believe they are not needed here.

Reply:
I am not sure, why do you think this. Can you elaborate?

And no. Those are are written the old school way.

Reply:
Yeah, sorry, I've missed that it would be nice to keep `i tak` together.